LICENSE.txt

```diff
@@ -0,0 +1,214 @@
+LICENSE
+
+VMware vmw-guestinfo
+
+Copyright � 2017 VMware, Inc.  All rights reserved.
+
+This product is licensed to you under the Apache License version 2.0 (the "License").  You may not use this product except in compliance with the License.
+
+                                 Apache License

```

Comment:
do we need that block? I'd prefer the LICENSE file to have just the standard content